scripts/asm-installer/install_asm

```diff
@@ -682,7 +684,11 @@ OPTIONS:
                                       Name of the ca in the GCP CAS service used to
                                       sign certificates in the format
                                       'projects/project_name/locations/ \
-                                       ca_region/certificateAuthorities/ca_name'
+                                      ca_region/certificateAuthorities/ca_name'
+  -r|--revision_name <REVISION NAME>  Custom revision label. Label needs to follow DNS
+                                      label formats (re: RFC 1123). Not supported if
+                                      control plane is managed. Recommended format:
+                                      asm-revisionTag
```

Comment:
I think maybe "Prefixing the revision name with 'asm' is recommended" is clearer than "Recommended format: asm-revisionTag". (plus tags aren't a thing yet)